knpu/facebook2.rst

```diff
@@ -385,14 +385,16 @@ a try-catch block::
         // ...
 
         try {
-            $json = $facebook->api('/me');
+            $json = $facebook->api('/me?fields=email,first_name,last_name');
         } catch (\FacebookApiException $e) {
             return $this->render('failed_token_request.twig', array('response' => $e->getMessage()));
         }
         var_dump($json);die;
         // ...
     }
 
+> Request fields you want explicitly due to Facebook API changes.
```

Comment:
How about:

> Due to recent Facebook API changes, you now need to add `?fields=` to explicitly
> ask for which fields you want.

AND, we can put it in a note structure - https://github.com/symfony/symfony-docs/blob/10ea5e9445ba9cf45e1cb98afe8977276af2ad6d/reference/forms/types/options/post_max_size_message.rst.inc#L10

I know, we don't work too much on the `.rst` format anymore :).

You should totally merge after making this change. Thanks!

P.S. you should have repo access now ;)